getconf/base.py

```diff
@@ -307,6 +161,64 @@ def gettimedelta(self, key, default='0d', doc=''):
         else:
             return datetime.timedelta(**{unit: value})
 
+
+class ConfigGetter(CustomConfigGetter):
```

Comment:
It feels weird that the canonical `ConfigGetter` class inherits from a custom class. Subclassing is usually the way to customize.

Reply:
Yup, I'd like to rename `ConfigGetter` as `LegacyConfigGetter` and `CustomConfigGetter` as `ConfigGetter`, maybe in a v2 but I wanted to keep these commits as refactor-only, not breeaking any existing code and without modifying any test.